crates/ruff/src/checkers/tokens.rs

```diff
@@ -60,6 +60,15 @@ pub fn check_tokens(
     ]);
     let enforce_extraneous_parenthesis = settings.rules.enabled(Rule::ExtraneousParentheses);
     let enforce_type_comment_in_stub = settings.rules.enabled(Rule::TypeCommentInStub);
+    let enforce_todos = settings.rules.any_enabled(&[
+        Rule::InvalidTODOTag,
+        Rule::TODOMissingAuthor,
+        Rule::TODOMissingColon,
+        Rule::TODOMissingText,
+        Rule::TODOMissingSpaceAfterColon,
+        // Rule::InvalidTODOCapitalization,
+        //     Rule::TODOMissingLink,
```

Comment:
As per the [naming convention](https://beta.ruff.rs/docs/contributing/#rule-naming-convention), I think these should be named `Missing[X]InTODO` (_allow missing author in TODO_) where `X` is the missing part.

Reply:
Changed!